### PR TITLE
add subview grant

### PR DESCRIPTION
**Pull request by icecream-teams**

### CODE REVIEW

File: grant/projects/subview.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+- Shell ( Front-end Engineer )
+- Eason ( DB Engineer )
+
+### **Legal Structure**
+
+Currently Subview has no defined legal structure other then a Dapp created by Subview team.
+
+## **Team Code Repos**
+
+The version is not open source.
```
Comment (Jin-Itering) [inline grant/projects/subview.md:28]: Our grant program has open-source requirement for what you apply for the grant

Comment (icecream-teams) [reply grant/projects/subview.md:28]: ok, We are preparing, and will submit the relevant grant code next

**Comment by Jin-Itering** [comment]
image

I just tried inspect a contract, I am not sure what happened, is it a problem? subscan has no such problem

**Comment by icecream-teams** [comment]
> image
> 
> I just tried inspect a contract, I am not sure what happened, is it a problem? subscan has no such problem

A problem with the latest update, which has been fixed

**Comment by Jin-Itering** [comment]
> > image
> > I just tried inspect a contract, I am not sure what happened, is it a problem? subscan has no such problem
> 
> A problem with the latest update, which has been fixed

Thanks. I just have a much slower experience than subscan, not sure where is the problem.

File: grant/projects/subview.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+## **Collaborative Grant Proposal and Current Development**
+
+### **Proposal**
+
+Subview provides a comprehensive, easy-to-use interface for users to view, confirm, and inspect transactions on EVM (Ethereum Virtual Machine).
+
+Subview will provide the following to Crab community:
+
+- crab.subview.xyz provides user access to crab network statistics
```
Comment (Jin-Itering) [inline grant/projects/subview.md:38]: "crab smart chain statistics"?

File: grant/projects/subview.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+- Full-featured API: GraphQL provides the opportunity to test API calls directly from a web interface, and RPC methods are well documented.
+
+### Subview **1.2**
+
+- DEX Info Support: Allow users to view the statistics of dex (similar to uniswap).
+
+## **Colabriation Funding**
+
+Total funding amount: 5,000 USDT.
```
Comment (Jin-Itering) [inline grant/projects/subview.md:70]: We generally prefer to half RING, half USDT, is that ok with you?

File: grant/projects/subview.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
+# subview.xyz
+
+## **Project Description**
+
+Subview is blockchain explorer for inspecting and analyzing EVM Chains. Subview provides a comprehensive, easy-to-use interface for users to view, confirm, and inspect transactions on EVM (Ethereum Virtual Machine).
+
+### **Why is this project is good for the Crab ecosystem:**
+
+1. Crab DVM currently does not have a specific DVM block explorer. For developers, will encounter obstacles when using the Crab network, especially when developers develop a Dapp. Subview can provide detailed block information and transaction information，developers can read and verify Solidity smart contracts and access pre-existing contracts to fast-track development.
```
Comment (Jin-Itering) [inline grant/projects/subview.md:9]: I see subscan also explore some CSC data, like: https://crab.subscan.io/account/0x0050f880c35c31c13bfd9cbb7d28aafaeca3abd2?tab=contract

So, what's the difference between you two?

File: grant/projects/subview.md (hunk 3 above)
Comment (icecream-teams) [reply grant/projects/subview.md]: Of course.

File: grant/projects/subview.md (hunk 4 above)
Comment (icecream-teams) [reply grant/projects/subview.md:9]: After reading the comparison, the contract analysis part is similar to the subview. Contract parsing is also the most basic part.

diff:
https://subview.xyz/tokens // Tokens can count the existence of all tokens in the crab smart chain, which is particularly useful.

https://subview.xyz/token/0x8eb2281bbD90ea671C9f6Cb1e51102B7FE3D480e/token-transfers // Comprehensive ERC20 token analysis

https://subview.xyz/token/0x3788df4fdC026f5Ea91a333fCf7CeD7a52c92471/token-transfers // ERC721 token analysis

File: grant/projects/subview.md (hunk 2 above)
Comment (icecream-teams) [reply grant/projects/subview.md]: yes, updated.

**Comment by icecream-teams** [comment]
> > > image
> > > I just tried inspect a contract, I am not sure what happened, is it a problem? subscan has no such problem
> > 
> > 
> > A problem with the latest update, which has been fixed
> 
> Thanks. I just have a much slower experience than subscan, not sure where is the problem.

There are not a lot of users now. We just accelerated the transaction details page and block details page that are used more, which can reduce a lot of server expenses.
If the users of the crab smart network increase in the later stage, we will improve the access speed and optimization.

File: grant/projects/subview.md (hunk 1 above)
Comment (icecream-teams) [reply grant/projects/subview.md:28]: https://github.com/SubviewOrg/subview-crab-chain

Comment (Jin-Itering) [reply grant/projects/subview.md:28]: Thanks. It seems lack of open source license file in the repo?

Comment (icecream-teams) [reply grant/projects/subview.md:28]: https://github.com/SubviewOrg/subview-crab-chain/blob/master/explorer/LICENSE

updated

Comment (1022ranji) [reply grant/projects/subview.md:28]: okay

**Comment by 1022ranji** [comment]
All functions in the scope have been tested, we'll send the tokens to the address that you supplied in md within one week.

**Review by 1022ranji — APPROVED** [review]

**Pull request merged**